backend/src/main/resources/application.yml

```
@@ -8,6 +8,9 @@ spring:
         order_inserts: true  # INSERT 순서 최적화
         order_updates: true  # UPDATE 순서 최적화
 
+server:
+  forward-headers-strategy: framework
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)